### PR TITLE
Docs - SYS query examples

### PR DESCRIPTION
**Pull request by petermarshallio**
Added two example queries on SYS tables and adjusted formatting of one that was already there

This PR has:
- [x] been self-reviewed.
- [x] been tested in a test Druid cluster

@2bethere I'd be grateful for a sanity check on the examples.
@techdocsmith

### CODE REVIEW

File: docs/querying/sql.md [hunk 1]
```diff
@@ -1169,6 +1179,26 @@ GROUP BY 1
 ORDER BY 2 DESC
 ```
 
+This query goes a step further and shows the overall profile of available, non-realtime segments across 1m row buckets for the `foo` datasource:
+
+```sql
+SELECT ABS("num_rows" /  1000000) as "bucket",
+  COUNT(*) as segments,
+  SUM("size") / 1048576 as totalSizeMb,
```
Comment (FrankChen021) [inline docs/querying/sql.md:1187]:
```suggestion
  SUM("size") / 1048576 as totalSizeMiB,
```

File: docs/querying/sql.md [hunk 2]
```diff
@@ -1169,6 +1179,26 @@ GROUP BY 1
...
+  MIN("size") / 1048576 as minSizeMb,
```
Comment (FrankChen021) [inline docs/querying/sql.md:1188]:
```suggestion
  MIN("size") / 1048576 as minSizeMiB,
```

File: docs/querying/sql.md [hunk 3]
```diff
@@ -1169,6 +1179,26 @@ GROUP BY 1
...
+  AVG("size") / 1048576 as averageSizeMb,
```
Comment (FrankChen021) [inline docs/querying/sql.md:1189]:
```suggestion
  AVG("size") / 1048576 as averageSizeMiB,
```

File: docs/querying/sql.md [hunk 4]
```diff
@@ -1169,6 +1179,26 @@ GROUP BY 1
...
+  MAX("size") / 1048576 as maxSizeMb,
```
Comment (FrankChen021) [inline docs/querying/sql.md:1190]:
```suggestion
  MAX("size") / 1048576 as maxSizeMiB,
```

**Review by FrankChen021 — COMMENTED** [review]
Overall LGTM with some suggestions.

**Comment by petermarshallio** [comment]
Thank you @FrankChen021

**Review by FrankChen021 — APPROVED** [review]

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**